src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -769,6 +774,52 @@ private static void DeleteCountOfRevisions(DocumentsOperationContext context, Sl
             }
         }
 
+        private void MarkRevisionsAsConflictedIfNeeded(DocumentsOperationContext context, Slice lowerId, Slice idSlice, DocumentFlags flags, TableValueReader tvr, Table table,
+            Slice changeVectorSlice)
+        {
+            // Revisions are immutable, but if there was a conflict we need to update the flags accordingly with the `Conflicted` flag.
```

Comment:
Why is that the case? The revision itself isn't conflicted, after all. 
Or is it that it was conflicted on another node?

Reply:
This is relevant in the case when revisions are enabled.
So node A generates a document with a revision, and node B also.
When the conflict is resolved, we need to add the `Conflicted` flag to an already existing revision.